capabilities/communication/email-templates/profile.supr

```diff
@@ -28,6 +37,37 @@ usecase GetTemplateContent safe {
     text
     html
   }
+
+  example Successful {
+    input {
+      id = "d-ef6d417300f74f96a9f722050edae85f"
+    }
+
+    result {
+      subject = "Station test"
+      text = "Station usecase automated test
+
+From: {{from}}
+to: {{to}}
+
+Unsubscribe ( {{{unsubscribe}}} ) - Unsubscribe Preferences ( {{{unsubscribe_preferences}}} )"
```

Comment:
@kysely I checked how it is rendered (and if :) ) in latest Air. And for this, highlighting seems to be a bit broken.

<img width="985" alt="image" src="https://user-images.githubusercontent.com/959390/143214458-523d74e6-2d81-4287-95a5-fa77d96681b1.png">

But example json seems to be valid. Do you think it is ok to merge?

Reply:
I know 😢 The highlighter is very naive (but lightweight) and there's an open discussion of whether we'll keep it or use a more robust one.

As for this case, I'd still merge the example now. Having it there feels more important than broken highlighting.

Reply:
👍 I still think your suggestions should be worked in, so I will do it and maybe it will work better with highlighter